app-emulation/aranym/aranym-1.0.2.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Atari Running on Any Machine"
+DESCRIPTION="
+ARAnyM (Atari Running on Any Machine) is a multiplatform virtual machine for \
+running Atari ST/TT/Falcon operating systems and applications.
+"
+HOMEPAGE="http://github.com/aranym/aranym"
+COPYRIGHT="2001-2018 ARAnyM developer team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aranym/aranym/archive/ARANYM_${portVersion//./_}.tar.gz"
+CHECKSUM_SHA256="c0935aee14cca55c08e75a94b77dcd69f326ae563ae371b22c7b9d8efd82e24e"
+SOURCE_DIR="aranym-ARANYM_${portVersion//./_}"
+PATCHES="aranym-$portVersion.patch"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	aranym = $portVersion
+	app:aranym = $portVersion
+	cmd:aranym = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libSDL_image_1.2
+	lib:libjpeg
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl_1.2
+	devel:libSDL_image_1.2
+	devel:libjpeg
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:make
+	cmd:gcc
+	cmd:sdl_config
+	"
+
+BUILD()
+{
+	rm -f config.guess config.sub
+	aclocal -I m4
+	autoconf
+	autoheader
+	automake --add-missing --copy || true
+	runConfigure ./configure --disable-sdl2
```

Comment:
```diff
-	runConfigure ./configure --disable-sdl2
+	runConfigure --omit-dirs binDir ./configure --bindir "$appsDir" \
+		--disable-sdl2
```
(Same suggestion for `aranym-999~git.recipe`.)

Reply:
....  --bindir=$appsDir ... ?

Reply:
I think both work. You can have either `--bindir=$appsDir` or `--bindir $appsDir`.
Maybe I should do some metrics :)

Reply:
Having either could cause confusion imho :) (but if so I stand corrected) :)

Reply:
It's `--bindir=$appsDir`. From `./configure --help`

      --bindir=DIR            user executables [EPREFIX/bin]